kgo.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+const foreign = require('foreign');
+const kgo = require('kgo');
+const S = require('sanctuary');
+const shuv = require('shuv');
+
+
+const readFile = (filename, callback) => fs.readFile(filename, {encoding: 'utf8'}, callback);
+const complete = (error, result) => {
+  if(error == null){
+    process.stdout.write(result);
+    process.exit(0);
+  }else{
+    process.stderr.write(String(error) + '\n');
+    process.exit(1);
+  }
+};
+
+const main = () => {
+  const relative = (filename) => path.join(process.argv[2], filename);
+
+  kgo
+  ('index', shuv(readFile, relative('index.txt')))
+  ('filePaths', ['index'], kgo.sync(index => S.lines(index).map(relative)))
+  ('files', ['filePaths'], shuv(foreign.parallel, readFile))
+  ('concated', ['files'], kgo.sync(strings => strings.join('')))
```

Comment:
I suggest requiring Ramda and replacing `strings => strings.join('')` with `R.join('')`.